pkg/cloud/aws/actuators/machine/utils_test.go

```diff
@@ -60,7 +60,7 @@ func TestProviderConfigFromMachine(t *testing.T) {
 			Namespace: "openshift-cluster-api",
 			Name:      "testClass",
 		},
-		ProviderConfig: *encodedProviderConfig.Value,
+		ProviderSpec: *encodedProviderConfig.Value,
```

Comment:
s/encodedProviderConfig/encodedProviderSpec.

Reply:
Again, bad variable naming. `encodedProviderConfig` is returned by `EncodeProviderConfig`, which returns `*clusterv1.ProviderSpec`. This PR is quite confusing due to the variable naming and fact `.providerConfig` is actualy `.providerSpec`.

Reply:
this has to stay, machine class has provider spec as `runtime.RawExtension`. Yeah.